frappe/public/js/frappe/form/tab.js

```diff
@@ -42,23 +42,21 @@ export default class Tab {
 			hide = true;
 		}
 
-		if (!hide && !this.df.show_dashboard) {
-			// show only if there is at least one visibe section or control
+		if (!hide) {
+			// show only if there is at least one visible section or control
 			hide = true;
 			if (
 				this.wrapper.find(
 					".form-section:not(.hide-control, .empty-section), .form-dashboard-section:not(.hide-control, .empty-section)"
 				).length
 			) {
 				hide = false;
+			} else if (this.df.show_dashboard && this.frm.is_new()) {
+				// hide if dashboard and not saved and no section/control visible
+				hide = true;
```

Comment:
this condition is moot? we set `hide = true` just above the if statement

Reply:
Removed the `else if` block